### PR TITLE
Add Display ID column to the distribution selector

### PR DESCRIPTION
**Pull request by olegrise**
@alex-deaconu please review. Staged at https://apps-stage-0.risevision.com

### CODE REVIEW

File: src/distribution-selector/distribution-list.html [hunk 1]
```diff
@@ -32,6 +35,7 @@ <h4 class="text-muted u_margin-md-bottom u_margin-md-top">No Displays Available<
         <tbody class="table-body">
           <tr class="table-body__row u_clickable display" ng-click="toggleDisplay(display.id);" ng-class="{'active' : isSelected(display.id) }" ng-repeat="display in displays.list">
             <td id="displayName-{{display.id}}" class="table-body__cell display-name"><span>{{display.name}}</span></td>
+            <td id="displayId-{{display.id}}" class="table-body__cell display-id"><span>{{display.id}}</span></td>
```
Comment (alex-deaconu) [inline src/distribution-selector/distribution-list.html]: @olegrise as you're using the `hidden-xs` in the <th>, when viewing at mobile resolutions the Header name disappears but the column persists.

I think you need to add the `hidden-xs` class to this new <td> as well, so they are both hidden on small resolutions.

Thanks!

**Comment by alex-deaconu** [comment]
@olegrise looks good, one comment regarding mobile resolutions (hidden-xs class). Wondering if you can also update the package.json file with the new version tag.

Thanks!

**Comment by olegrise** [comment]
@alex-deaconu thanks for the review. Good points. All is fixed.

**Comment by alex-deaconu** [comment]
@olegrise I don't believe the `dist` file contains the latest change. Can you please confirm? Thanks.

**Comment by alex-deaconu** [comment]
@olegrise 👍 thanks for fixing!

**Comment by olegrise** [comment]
@alex-deaconu thank you for thorough review!

**Pull request merged**